pages/api/word-list/[user].js

```diff
@@ -0,0 +1,37 @@
+import dbConnect from "@/db/connect";
+import Entry from "@/db/models/Entry";
+
+export default async function handler(request, response) {
+  const { user } = await request.query;
+  await dbConnect();
+
+  // Should I put something in case the user has not been loaded yet?
+  // It always takes the session a bit to load in the active user.
+  // I got a lot of error messages when I left the value as just undefined. So I changed it to "loading"
+  // Something like the following?
+  // if (user === "loading") {
+  //   response.status(102).json({ status: "processing" });
+  // }
+
+  if (user !== "loading") {
```

Comment:
you could also use an early return here to prevent nested if statement. then you could end the handler function if the user === 'loading'